api/v1/standalone-dns-proxy/standalone-dns-proxy.proto

```diff
@@ -0,0 +1,55 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+syntax = "proto3";
+
+package standalonednsproxy;
+
+option go_package = "github.com/cilium/cilium/api/v1/standalonednsproxy";
+
+service FQDNData {
+  rpc SubscribeToDNSPolicies(stream  DNSPoliciesResult) returns (stream DNSPolicies) {}
+  rpc UpdatesMappings(FQDNMapping) returns (UpdatesMappingsResult) {}
```

Comment:
nit: `UpdateMapping` would be better, in my opinion. Each call is concerned with one specific mapping.

If we want to apply Google's gRPC API guidance, we'd call the argument `UpdateMappingRequest` and the return value `UpdateMappingResponse`, which gives us more freedom to transform the API after shipping it - extending these messages is backward-compatible in protobuf. However, doing so in the current naming would likely be confusing.

Given the above RPC is a bidrectional stream, why is this not a bi-directional stream of mappings and ACKs?

Reply:
UpdateMapping is called by SDP agent on receiving update from dnsserver and felt grpc unary would suit this request and bidirectional stream is not required.